src/modules/deviceMethods.ts

```diff
@@ -16,4 +16,9 @@ export const deviceMethods = {
     getUserDevices: (id: string): Promise<UserDeviceInfo[]> => {
         return prisma.userDevice.findMany({ where: { userId: id }, include: { device: true } });
     },
+    deleteDevices: (data: DeleteDevice) => {
+        return prisma.userDevice.delete({
+            where: { deviceName_deviceId: { deviceName: data.deviceName, deviceId: data.deviceId } },
```

Comment:
you only need Id probably

Reply:
In scheme UserDevices @@unique([deviceName, deviceId]). These`re required fields